astartes/samplers/interpolation/kennardstone.py

```diff
@@ -1,15 +1,16 @@
 from astartes.samplers import AbstractSampler
 
 # https://github.com/yu9824/kennard_stone
-from kennard_stone import train_test_split
+from kennard_stone import train_test_split as ks_train_test_split
+
+import numpy as np
```

Comment:
I typically use a different order for imports e.g. placing standard libraries like numpy first, then related 3rd party imports like KS, then local imports from astartes. Just curious, what guidelines would you like to follow for this repo?

Reply:
Agreed that we should settle on a configuration -- let's go with [`isort`](https://pycqa.github.io/isort/) which integrates nicely with the `black` formatter. I will open a separate PR to edit the action and update all the files.